casper/src/main/resources/CoopSecurityCheck.rho

```diff
@@ -0,0 +1,31 @@
+new 
```

Comment:
Sorry, I forgot that this contract is used inside Registry also so it cannot be defined in a separate file, just like TreeHashMap.